tests/unit/HttpRequestTest.php

```diff
@@ -270,9 +270,9 @@ public function testHttpRequestCurlAdapter()
             )
         );
         $response = $request->get();
-        $this->assertEquals(
+        $this->assertStringContainsString(
             sprintf(
-                'Failed to connect to localhost port %d after 0 ms: Connection refused',
+                'Connection refused',
```

Comment:
You forgot the port

Reply:
I guess that we need to assert 2 times, first one for Failt to connect to localhost port %d and second on is Connection refusued. Or another option is use StartWith and EndWith